From f8e158bc72986e46b93d05358c29db0c10f2fe9f Mon Sep 17 00:00:00 2001
From: Karlchen <k_straussberger@netzland.net>
Date: Wed, 17 Sep 2025 21:18:49 +0200
Subject: [PATCH] Update to fmt 12.0.0

Signed-off-by: Michael Nosthoff <buildroot@heine.tech>
Upstream: https://github.com/gerbera/gerbera/commit/f8e158bc72986e46b93d05358c29db0c10f2fe9f
[buildroot@heine.tech: removed all patch content not patching actual code]

---
 src/web/config_load.cc            | 4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/src/web/config_load.cc b/src/web/config_load.cc
index 8b296c43e0..4259ae3331 100644
--- a/src/web/config_load.cc
+++ b/src/web/config_load.cc
@@ -150,7 +150,11 @@ void Web::ConfigLoad::addNewValue(
 template <typename T>
 void Web::ConfigLoad::setValue(Json::Value& item, const T& value)
 {
+#if FMT_VERSION >= 120000
+    static_assert(fmt::is_formattable<T, fmt::format_context>::value, "T must be formattable");
+#else
     static_assert(fmt::has_formatter<T, fmt::format_context>::value, "T must be formattable");
+#endif
     item[CONFIG_LOAD_VALUE] = fmt::to_string(value);
 }
 
